api/app/schemas/morecast_v2.py

```diff
@@ -141,6 +143,14 @@ class IndeterminateDailiesResponse(BaseModel):
     forecasts: List[WeatherIndeterminate]
 
 
+class SimulateIndeterminateIndices(BaseModel):
+    simulate_records: List[WeatherIndeterminate]
+
+
+class SimulatedWeatherIndeterminateResponse(BaseModel):
+    simulatedForecasts: List[WeatherIndeterminate]
```

Comment:
nit: Maybe change `simulatedForecasts` to snake case

Reply:
And when you do, you'll have to change it on the frontend when it's looked up from the response.